op-node/cmd/genesis/cmd.go

```diff
@@ -49,39 +51,36 @@ var Subcommands = cli.Commands{
 				return err
 			}
 
-			// Add the developer L1 addresses to the config
-			if err := config.InitDeveloperDeployedAddresses(); err != nil {
-				return err
+			var deployments *genesis.L1Deployments
+			if l1Deployments := ctx.String("l1-deployments"); l1Deployments != "" {
+				deployments, err = genesis.NewL1Deployments(l1Deployments)
+				if err != nil {
+					return err
+				}
 			}
 
-			if err := config.Check(); err != nil {
-				return err
+			if deployments != nil {
+				config.SetDeployments(deployments)
 			}
 
-			l1Genesis, err := genesis.BuildL1DeveloperGenesis(config)
-			if err != nil {
-				return err
+			if err := config.Check(); err != nil {
+				return fmt.Errorf("deploy config at %s invalid: %w", deployConfig, err)
 			}
```

Comment:
This can be moved up to immediately after the config is created right?

Reply:
It needs to happen after the deployments are set, otherwise it will fail

Reply:
Oops missed that, thanks!